src/spikeinterface/extractors/tests/common_tests.py

```diff
@@ -117,12 +117,26 @@ def test_open(self):
                 kwargs = entity
                 sorting = self.ExtractorClass(**kwargs)
 
-            num_seg = sorting.get_num_segments()
+            num_segments = sorting.get_num_segments()
             unit_ids = sorting.unit_ids
+            sampling_frequency = sorting.get_sampling_frequency()
 
-            for segment_index in range(num_seg):
+            for segment_index in range(num_segments):
                 for unit_id in unit_ids:
-                    st = sorting.get_unit_spike_train(segment_index=segment_index, unit_id=unit_id)
+                    # Test that spike train has the propert units
+                    spike_train = sorting.get_unit_spike_train(segment_index=segment_index, unit_id=unit_id)
+                    sample_differences = np.diff(spike_train)
+                    no_spike_firing_in_the_same_frame = np.all(sample_differences > 0)
```

Comment:
I am not sure that all formats ensure this.
Some sorting could make mistakes and return 2 spikes at the same sample index not ?

Reply:
That could be true, but I am not too concerned. This code is meant only to be used with the data in our testing suite as a heuristic for proper time scaling. The code does not have too be valid for all the universe of possible spike sorting configurations (where your hypotethical is included).  Thi should  only detect changes in the code within our testing suite so we are alerted that the representation was broken.

Maybe there is a better way. I will think about it. Do you have a suggestion of how to better test this?